flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing

Reply:
yes, we'll try to support arm build in future iteration

Reply:
how about building from source? I thought that the goal of using one of these "cross-platform" thingy was to be cross platform.

Reply:
https://github.com/flathub/flathub/pull/5729#discussion_r1804621841